frontend/src/modules/Misc/views/Glossaries/GlossaryManagement.js

```diff
@@ -227,7 +224,7 @@ const GlossaryManagement = (props) => {
             className={classes.root}
             defaultExpanded={['3']}
             defaultCollapseIcon={<ArrowDropDownIcon />}
-            defaultExpandIcon={<ArrowRightIcon />}
+            defaultExpandIcon={<ArrowRight />}
```

Comment:
I wonder why we changed it? Is this import equivalent to the previous?
I see that react docs suggest to import this as `import ArrowRightIcon from '@mui/icons-material/ArrowRight';` 
Just to check if this is an intentional change

Reply:
Since this is a default export, we can assign any name we want upon importing it, so the name itself does not matter as long as you are importing it correctly.

That said, I have reverted it to the previous name to be more consistent with other icons import, and to be more descriptive.